src/engine/enginebuffer.cpp

```diff
@@ -1179,11 +1180,6 @@ void EngineBuffer::postProcess(const int iBufferSize) {
 }
 
 void EngineBuffer::updateIndicators(double speed, int iBufferSize) {
-    if (!m_trackSampleRateOld || !m_trackSamplesOld) {
```

Comment:
Wasn't this guarding against a divide by zero below?

Reply:
The function is not called in this case. But I have added it back as VERIFY_OR_DEBUG_ASSERT() as an extra savety net.

Reply:
Ah I didn't notice, thanks.